zipline/pipeline/loaders/equity_pricing_loader.py

```diff
@@ -166,7 +166,7 @@ def _inplace_currency_convert(self, columns, arrays, dates, sids):
             by_spec[column.currency_conversion].append(array)
 
         # Nothing to do for terms with no currency conversion.
-        by_spec.pop(None)
+        by_spec.pop(None, None)
```

Comment:
This is the bugfix. The code above this groups the columns by "conversion spec", which describes what currency conversion needs to happen on the column. `None` is used for columns that don't need conversion, and we want to remove those before proceeding, since we don't need to do anything for those. All my previous tests that exercised currency conversion also included at least one non-converted column, so the `pop(None)` always succeeded. When running with all converted columns, however, we crash here because there's no `None` key.